src/main/resources/crowdin.yml

```diff
@@ -124,4 +124,22 @@ files: [
   #
   # "scheme" : "",
  }
+]
+
+targets: [
```

Comment:
This `crowidn.yml` file is supposed to showcase the most important configurations for a quick start. We would like not to add this `targets` configuration here since this option is not widely used and is needed only in some specific cases.

If it's needed for tests, there should be a way to use some testing configuration file instead of this file.

Reply:
Since this file related only to tests, It would be better to move it to the [test resources](https://github.com/crowdin/crowdin-cli/tree/cli3/src/test/resources)